scim-core/src/test/java/org/apache/directory/scim/core/repository/PatchHandlerTest.java

```diff
@@ -19,6 +19,16 @@
 
 package org.apache.directory.scim.core.repository;
 
+import static java.util.Map.entry;
```

Comment:
My idea is probably set up with some linters that automatically re-ordered this, I can undo it if you would like!

Reply:
It's fine, it's isolated to a single test class so it's not the end of the world.
But if you want to clean it up or rewrite/squash any of the go for it.

If not, i'll try to merge this later today!